app/graphql/types/query_type.rb

```diff
@@ -0,0 +1,30 @@
+module Types
+  class QueryType < Types::BaseObject
+
+    field :questions, [QuestionType], null: false
+ 
+    field :surveys, [SurveyType], null: false
+
+    field :all_questions_by_survey, [QuestionType], null: false do
+      argument :id, ID, required: true
+    end
+
+   
+
+    def questions
+      RatingQuestion.all
+   
+    end
+
+    def all_questions_by_survey(id:)
+      survey = Survey.find(id)
+      found_survey_id = survey.id.to_s
+      questions_per_survey = RatingQuestion.where(survey_id:found_survey_id)
```

Comment:
This variable is returned on the next line. You could collapse the final line of this method to be just this `RatingQuestion.where...` part of this line.

Reply:
Looking at it again, you could probably do `survey.rating_questions`; accessing the rating questions for the survey through the association method.